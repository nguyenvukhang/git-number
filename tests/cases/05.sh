# unstaged modified file (number)

init 3
echo "_" >file_3
_gitnu status
_gitnu add 3 # use number on file_3
_gitnu commit -m "first"
echo "_" >>file_3
_gitnu status
_gitnu add 1 # use number on file_3 again
save gitnu status
