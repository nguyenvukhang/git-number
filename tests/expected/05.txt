On branch main
Changes to be committed:
1	[32mnew file:   file_0001[m

Untracked files:
2	[31mfile_0002[m
3	[31mfile_3[m

